maintenance/passport-azure-ad/package.json

```diff
@@ -45,7 +45,6 @@
     "node-jose": "^2.0.0",
     "oauth": "0.9.15",
     "passport": "^0.4.1",
-    "request": "^2.72.0",
     "valid-url": "^1.0.6"
   },
```

Comment:
Can you also bump the version to 4.3.3, and add a changelog entry?

Reply:
Done